src/api/app/models/distribution.rb

```diff
@@ -38,8 +38,7 @@ def self.all_including_remotes
     list = Distribution.all_as_hash
     repositories = list.map{ |d| d['reponame'] }
 
-    remote_projects = Project.where("NOT ISNULL(projects.remoteurl)")
```

Comment:
I don't understand? Why should I remove it? We need it to select the remote projects... And no it won't fail because I never reach line 45 then...